.github/workflows/runtests.yml

```diff
@@ -64,7 +64,7 @@ jobs:
         with:
           python-version: '3.11'
       - run: python -m pip install --upgrade pip
-      - run: pip install mpmath mypy hypothesis
+      - run: pip install mpmath mypy hypothesis lark
```

Comment:
The test should only run for optional dependencies

Reply:
The test was failing without this, and I'm not sure. Do you have any idea what else I can do to solve the issue? It was complaining that it could not find the `lark` module.

Reply:
I already see the lark is there, but I'm not sure about the problem though.
I just request you to remove the modifications here, such that the CI can pinpoint to the log.

Reply:
Alright, I'll do that.

Reply:
I've removed all the changes I made related to `runtests.yml`. You should be able to check the CI logs for the issues now.